DevoxxClientMobile/src/main/java/com/devoxx/service/DevoxxService.java

```diff
@@ -1019,6 +1050,42 @@ private String findConferenceIdFromString(String fileContent) {
         }
         return "";
     }
+    
+    private static boolean isReloadNotification(String fileContent) {
+        try {
```

Comment:
Maybe we could parse the string (it has json format) with JsonParser and StringReader?